site/docs/guides/migration.md

```diff
@@ -76,28 +123,46 @@ Exported Nessie repository, 100 commits into 1 files, 1 named references into 1
 Once the
 export process has completed, the ZIP file will be available at `/tmp/export-<date>.zip`.
 
+### Import
+
 Download the `nessie-server-admin-tool-x.y.z-runner.jar` corresponding to the **new** Nessie version, here
 we assume version {{ versions.nessie }} (note: older versions have this tool under the name of `nessie-quarkus-cli`):
 
-```bash
-curl -L https://github.com/projectnessie/nessie/releases/download/nessie-{{ versions.nessie }}/nessie-server-admin-tool-{{ versions.nessie }}-runner.jar -o nessie-server-admin-tool-{{ versions.nessie }}-runner.jar
-```
-
-Create the MongoDB database to host the new Nessie repository. In this example we assume that the 
-MongoDB database is called `nessie` and is hosted on the same MongoDB instance as the legacy Nessie
+Create/setup the database to host the new target Nessie repository. In this example we assume that
+a MongoDB database is called `nessie` and is hosted on the same MongoDB instance as the source Nessie
 repository.
 
+Refer to the [Server Configuration](/nessie-latest/configuration/) reference docs for the respective
+Nessie version.
+
 Import the ZIP file into the new Nessie repository:
 
-```bash
-java \
-  -Dnessie.version.store.type=MONGODB \
-  -Dquarkus.mongodb.database=nessie \
-  -Dquarkus.mongodb.connection-string=mongodb://nessie.example.com:27017 \
-  -jar nessie-server-admin-tool-{{ versions.nessie }}-runner.jar import \
-  --path "/tmp/export-$(date +'%Y-%d-%m').zip" \
-  --commit-batch-size 1000
-```
+See [Nessie Server Admin tool reference docs](/nessie-latest/export_import) for details.
+
+=== "Docker Image"
+    The Docker image is available for Nessie Server admin tool since version 0.83.1.
+    ```bash
+    docker run --rm -ti quay.io/projectnessie/nessie-server-admin:{{ versions.nessie }} \
+      import \
+      --path "/tmp/export-$(date +'%Y-%d-%m').zip" \
```

Comment:
This won't be the same container as the one that did the export, will it? I think we need to either:

1. Mount this path on the host
2. Give a name to the container with `--name whatever` and remove the `--rm` switch

Reply:
I went with a different approach, but also moved the `Standalone Jar` to the front.
Using a Docker container it probably much painful than just running the jar, given that users might also want to use `application.properties`.